config/client.webpack.config.dev.js

```diff
@@ -0,0 +1,41 @@
+const path = require("path");
+const webpack = require("webpack");
+
+module.exports = {
+  name: "client",
+  target: "web",
+  devtool: "eval-source-map",
+  entry: [
+    "babel-polyfill",
+    "webpack-hot-middleware/client?path=/__webpack_hmr&timeout=20000&reload=false&quiet=false&noInfo=false",
+    path.resolve(__dirname, "../src/index.js")
+  ],
+  output: {
+    filename: "[name].js",
+    chunkFilename: "[name].bundle.js",
+    path: path.resolve(__dirname, "../dist/client"),
+    publicPath: "/static/"
```

Comment:
this should be `public`

Reply:
I don't believe so: https://webpack.js.org/configuration/output/#output-publicpath